misc/acrn-config/scenario_config/pci_dev_c.py

```diff
@@ -5,16 +5,43 @@
 
 import common
 import scenario_cfg_lib
+import board_cfg_lib
 
 PCI_DEV_TYPE = ['PCI_DEV_TYPE_HVEMUL', 'PCI_DEV_TYPE_PTDEV']
 
+def add_instance_to_name(i_cnt, bdf, bar_attr):
+    if i_cnt == 0 and bar_attr.name.upper() == "HOST BRIDGE":
+        tmp_sub_name = "_".join(bar_attr.name.split()).upper()
+    else:
+        if '-' in bar_attr.name:
+            tmp_sub_name = common.undline_name(bar_attr.name) + "_" + str(i_cnt)
+        else:
+            tmp_sub_name = "_".join(bar_attr.name.split()).upper() + "_" + str(i_cnt)
+
+    board_cfg_lib.PCI_DEV_BAR_DESC.pci_dev_dic[bdf].name_w_i_cnt = tmp_sub_name
 
 def generate_file(vm_info, config):
     """
     Generate pci_dev.c while logical_partition scenario
```

Comment:
Hi Sainath,
Thanks for your fix.
Could you please help to remove the phrase of " while logical_partition scenario".
Since acrn-config will not do any hard code on specific scenario. Any scenario which defines pci_devs in its pre-launched VM will have a pci_dev.c

Reply:
Done